chutney/server/src/main/java/com/chutneytesting/campaign/api/CampaignController.java

```diff
@@ -152,4 +155,9 @@ public List<CampaignDto> getCampaignsByScenarioId(@PathVariable("scenarioId") St
             .collect(Collectors.toList());
     }
 
+    private void checkPresenceEnvironment(CampaignDto campaign) {
```

Comment:
It looks weird for me.
May be hasEnvironment(..) or isEnvironmentSetForCampaign()